driver/test/client_utils.h

```diff
@@ -2,14 +2,15 @@
 
 #include "driver/platform/platform.h"
 #include "driver/type_info.h"
+#include "driver/utils/unicode_conv.h"
 
 namespace {
 
     inline std::string extract_diagnostics(SQLHANDLE handle, SQLSMALLINT type) {
         std::string result;
         SQLSMALLINT i = 0;
         SQLINTEGER native = 0;
-        SQLTCHAR state[7] = {};
+        SQLTCHAR state[6] = {};
```

Comment:
Could you please expain this magic number in comments?

Reply:
Sure.